example/src/main/java/com/appunite/rx/example/CreatePostActivity.java

```diff
@@ -0,0 +1,129 @@
+    @InjectView(R.id.create_post_body_text)
+    EditText bodyText;
+    @InjectView(R.id.create_post_name_text)
+    EditText nameText;
+    @InjectView(R.id.create_post_activity_error)
+    TextView error;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.create_post_activity);
+
+        ButterKnife.inject(this);
+
+    //    toolbar.setNavigationIcon(R.drawable.ic_cancel_white_24dp);
+
+        final CreatePostPresenter createPostPresenter = new CreatePostPresenter(AndroidSchedulers.mainThread(),
+                FakeDagger.getPostsDaoInstance(getApplicationContext()));
+
+
+        MoreViewObservables.navigationClick(toolbar)
+                .compose(lifecycleMainObservable.bindLifecycle())
+                .subscribe(createPostPresenter.navigationClickObserver());
+
+        createPostPresenter.closeActivityObservable()
+                .compose(lifecycleMainObservable.bindLifecycle())
+                .subscribe(new Action1<Object>() {
+                    @Override
+                    public void call(Object o) {
+                        finish();
+                    }
+                });
+
+        ViewObservable.clicks(acceptButton)
+                .map(new Func1<OnClickEvent, Object>() {
+                    @Override
+                    public Object call(OnClickEvent onClickEvent) {
+                        return new Object();
+                    }
+                })
+                .compose(lifecycleMainObservable.bindLifecycle())
+                .subscribe(createPostPresenter.sendObservable());
+
+        WidgetObservable.text(bodyText)
+                .map(new OnTextChangeAction())
+                .compose(lifecycleMainObservable.<String>bindLifecycle())
+                .subscribe(createPostPresenter.bodyObservable());
+
+        WidgetObservable.text(nameText)
+                .map(new OnTextChangeAction())
+                .compose(lifecycleMainObservable.<String>bindLifecycle())
+                .subscribe(createPostPresenter.nameObservable());
+
+        createPostPresenter.finishActivityObservable()
+                .compose(lifecycleMainObservable.<Response>bindLifecycle())
+                .subscribe(new Action1<Response>() {
+                    @Override
+                    public void call(Response response) {
+                        finish();
+                    }
+                });
+
+        createPostPresenter.postErrorObservable()
+                .compose(lifecycleMainObservable.<Throwable>bindLifecycle())
+                .subscribe(new Action1<Throwable>() {
+                    @Override
+                    public void call(Throwable throwable) {
+                        Toast.makeText(getApplicationContext(), R.string.create_post_error_message, Toast.LENGTH_SHORT).show();
+                    }
+                });
+
+        createPostPresenter.errorObservable()
```

Comment:
You should not display persistent errors for one time actions.
User click submit - if error will be displayed it should be dismissable after time or by user. Btw you display same error twice